diff --git a/node_modules/react-remove-scroll/dist/es2015/SideEffect.js b/node_modules/react-remove-scroll/dist/es2015/SideEffect.js
index 625c185..25dca05 100644
--- a/node_modules/react-remove-scroll/dist/es2015/SideEffect.js
+++ b/node_modules/react-remove-scroll/dist/es2015/SideEffect.js
@@ -15,6 +15,19 @@ var deltaCompare = function (x, y) { return x[0] === y[0] && x[1] === y[1]; };
 var generateStyle = function (id) { return "\n  .block-interactivity-".concat(id, " {pointer-events: none;}\n  .allow-interactivity-").concat(id, " {pointer-events: all;}\n"); };
 var idCounter = 0;
 var lockStack = [];
+// https://stackoverflow.com/questions/9038625/detect-if-device-is-ios/9039885#9039885
+var isIOS = typeof window !== 'undefined' && (
+  [
+    'iPad Simulator',
+    'iPhone Simulator',
+    'iPod Simulator',
+    'iPad',
+    'iPhone',
+    'iPod',
+  ].includes(navigator.platform) ||
+  // iPad on iOS 13 detection
+  (navigator.userAgent.includes('Mac') && 'ontouchend' in document)
+)
 export function RemoveScrollSideCar(props) {
     var shouldPreventQueue = React.useRef([]);
     var touchStartRef = React.useRef([0, 0]);
@@ -54,7 +67,12 @@ export function RemoveScrollSideCar(props) {
         }
         var canBeScrolledInMainDirection = locationCouldBeScrolled(moveDirection, target);
         if (!canBeScrolledInMainDirection) {
+          // Breaks Apple Pencil and breaks selection editing on iPadOs and iOS
+          // https://github.com/theKashey/react-remove-scroll/issues/101
+          // https://github.com/theKashey/react-remove-scroll/issues/81
+          if (event.type != 'touchmove' || !isIOS) {
             return true;
+          }
         }
         if (canBeScrolledInMainDirection) {
             currentAxis = moveDirection;
diff --git a/node_modules/react-remove-scroll/dist/es2019/SideEffect.js b/node_modules/react-remove-scroll/dist/es2019/SideEffect.js
index d654202..3fd1ca9 100644
--- a/node_modules/react-remove-scroll/dist/es2019/SideEffect.js
+++ b/node_modules/react-remove-scroll/dist/es2019/SideEffect.js
@@ -13,6 +13,19 @@ const generateStyle = (id) => `
 `;
 let idCounter = 0;
 let lockStack = [];
+// https://stackoverflow.com/questions/9038625/detect-if-device-is-ios/9039885#9039885
+const isIOS = typeof window !== 'undefined' && (
+  [
+    'iPad Simulator',
+    'iPhone Simulator',
+    'iPod Simulator',
+    'iPad',
+    'iPhone',
+    'iPod',
+  ].includes(navigator.platform) ||
+  // iPad on iOS 13 detection
+  (navigator.userAgent.includes('Mac') && 'ontouchend' in document)
+)
 export function RemoveScrollSideCar(props) {
     const shouldPreventQueue = React.useRef([]);
     const touchStartRef = React.useRef([0, 0]);
@@ -52,7 +65,12 @@ export function RemoveScrollSideCar(props) {
         }
         let canBeScrolledInMainDirection = locationCouldBeScrolled(moveDirection, target);
         if (!canBeScrolledInMainDirection) {
+          // Breaks Apple Pencil and breaks selection editing on iPadOs and iOS
+          // https://github.com/theKashey/react-remove-scroll/issues/101
+          // https://github.com/theKashey/react-remove-scroll/issues/81
+          if (event.type != 'touchmove' || !isIOS) {
             return true;
+          }
         }
         if (canBeScrolledInMainDirection) {
             currentAxis = moveDirection;
